content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning.md

```diff
@@ -274,6 +274,87 @@ version sort order is `v1`, followed by `v1beta1`. This causes the kubectl
 command to use `v1` as the default version unless the provided object specifies
 the version.
 
+### Version deprecation
+
+Starting in v1.19, a CustomResourceDefinition can indicate a version is deprecated.
+When API requests to a deprecated version are made, a warning message is returned in the API response as a header.
+The warning message for each deprecated version can be customized if desired.
```

Comment:
These are deprecated versions for a particular CR, right?  For instance, I could have foo.v1beta1.group and bar.v1beta1.group and I can choose to only deprecate foo.v1beta.group not the entire group/v1beta1.

Reply:
correct

Reply:
updated/clarified